README.md

```diff
@@ -22,6 +22,14 @@ mkdir -p "~/.terraform.d/plugins/$(go env GOHOSTOS)_$(go env GOHOSTARCH)/"
 cp ${GOPATH}/bin/terraform-provider-ansiblevault "~/.terraform.d/plugins/$(go env GOHOSTOS)_$(go env GOHOSTARCH)/"
 ```
 
+or
+
+In the repository
+```bash
+make build
+make install
+```
+
 Without golang
```

Comment:
Peut-être transformer en titre de sous-partie ici et au niveau de `If you have Golang installed` pour mieux séparer les méthodes d'installation avec ou sans Go

```suggestion
### Without golang
```

Reply:
On peut. J'accepte pas ta modification car ça fait un commit non signé et après ça va faire le bordel.

Reply:
done